apps/site/src/pages/api/external-service-method.api.ts

```diff
@@ -45,8 +45,22 @@ export default createApiKeyRequiredHandler<
     } catch (error) {
       const axiosError = error as AxiosError;
 
-      const { status, data } = axiosError.response ?? {};
+      const { status, data } = axiosError.response ?? {
+        status: 500,
+        data: { errors: [{ message: "An unknown error occurred." }] },
+      };
 
-      res.status(status ?? 500).json((data as any) ?? {});
+      res.status(status ?? 500).json(
+        formatErrors({
+          msg:
+            data &&
+            typeof data === "object" &&
+            "errors" in data &&
+            (data.errors as any[])?.[0].message
+              ? (data.errors as any[])?.[0].message
+              : "An unknown error occurred.",
+          code: status.toString(),
+        }),
```

Comment:
If this is guaranteed to be backend code, do we want to log the error using `util.inspect` here? Slightly concerned about us having absolutely no information if it doesn't match this weird type you're checking against here.

Reply:
We could `Sentry.captureException` here (this weird type is Axios which I'd love to get rid of)

Reply:
As long as it won't fail on circular references (which is why I suggested `util.inspect`), we've ran into a lot of issues with JSON.stringifying an axios error